.github/workflows/release.yml

```diff
@@ -19,64 +19,72 @@ jobs:
     env:
       LERNA_DEFAULT_ARGS: --exact --concurrency 1 --yes --conventional-commits --create-release github --allow-branch "*"
     steps:
-      - name: Checkout Code
+      - name: Checkout code
         uses: actions/checkout@v2.3.5
-
-      - run: |
-          git fetch --prune --unshallow
+        with:
+          fetch-depth: 0
+          persist-credentials: false
```

Comment:
Will not persist the standard Github credentials since we'll be using our own from @side-ci-bot.